File: app/src/main/java/com/govnokoder/velotracker/services/MyReceiver.java
package com.govnokoder.velotracker.services;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import com.govnokoder.velotracker.BL.ParcelableTraining;

public class MyReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        ParcelableTraining parcelableTraining = intent.getParcelableExtra(LocationService.EXTRA_PARCELABLE_TRAINING);
        if (parcelableTraining != null) {
        }
    }
}

File: app/src/main/java/com/coursework/velotracker/Services/MyReceiver.kt
package com.coursework.velotracker.Services

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import com.coursework.velotracker.BL.Model.ParcelableTraining

class MyReceiver: BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        val parcelableTraining: ParcelableTraining? = intent.getParcelableExtra(LocationService.EXTRA_PARCELABLE_TRAINING)
    }
}